docs/azure/aksextensions.md

```diff
@@ -0,0 +1,127 @@
+
+With the extension, you can quickly develop, deploy Azure Kubernetes Diagnostics health-checks, run diagnosics tools to collect cluster logs like aks-periscope, create GitHub Starter workflows or deploy services like azure service operator in jsut one-click.
+
+To install the Azure Kuberentes Services Visual Code extension, open the Extensions view (`kb(workbench.view.extensions)`) and search for "AKS". Select the Microsoft [Azure Kuberentes Services](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) extension.
+
+![Install Azure Kuberentes Services](images/aksextensions/install-aks-extension.png)
+
+## Installation
+
+1. Download and install the [Azure Kubernetes Service extension](https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-aks-tools) for Visual Studio Code.
+
+2. Wait for the extension to finish installing then reload Visual Studio Code when prompted.
+
+3. Once the installation is complete, you'll see a section named **Azure** under **Clouds**.
+
+4. Sign in to your Azure Account by clicking **Sign in to Azure…**
+
+![Sign in to your Azure Account](images/aksextensions/Sign-in.png)
+
+## Features
+
+Once you successfully log in with your Azure Account, you can view all AKS clusters in your Azure subscriptions(s) under the section named **Azure**. You can right click on your AKS cluster and click a menu item to perform following actions.
+
+![Cloud explorer extension menu](images/aksextensions/right-click-menu.png)
+
+![Cloud explorer extension menu](images/aksextensions/right-click-menu-workflow.png)
+
+### Merge into Kubeconfig
+
+Right click on your AKS cluster and click on **Merge into Kubeconfig** to add the selected AKS cluster to the kubeconfig file.
+
+### Save Kubeconfig
+
+Right click on your AKS cluster and click on **Save Kubeconfig** to save the kubeconfig of the selected AKS cluster.
+
+### AKS Diagnostics
+
+Right click on your AKS cluster and click on **Run AKS Diagnostics** to display diagnostics information based on your AKS cluster's backend telemetry for:
+
+- Best Practices
+- Create, Upgrade, Delete and Scale issues
+- Identity and Security
+- Network and Connectivity issues
+- Node Health
+
+To perform further checks on your AKS cluster to troubleshoot and get recommended solutions, click on the AKS Diagnostics link at the top of the page to open it for the selected cluster. For more information on AKS Diagnostics, visit [AKS Diagnostics Overview](https://docs.microsoft.com/azure/aks/concepts-diagnostics).
+
+![AKS Diagnostics Webview](images/aksextensions/aks-diagnostics-webview.png)
+
+### AKS Periscope
+
+Right click on your AKS cluster and click on **Run AKS Periscope** to collect node and pod logs for your AKS cluster and to export them to an Azure storage account. Upon selecting the option, a web view will load providing you the option to generate a downloadable link for the collected logs as well as a shareable link with 7-day expiry.
```

Comment:
Rather than specifying exactly what Periscope collects, since it's more than logs and may change, how about starting with:

> Right click on your AKS cluster and click on **Run AKS Periscope** to extract detailed diagnostic information from your AKS cluster and export it to an Azure storage account.

?